src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisObserver.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.securityanalysis.server.service;
+
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.NonNull;
+import org.springframework.stereotype.Service;
+
+/**
+* @author Kevin Le Saulnier <kevin.lesaulnier at rte-france.com>
+*/
+@Service
+public class SecurityAnalysisObserver {
+    private final ObservationRegistry observationRegistry;
+
+    private static final String OBSERVATION_PREFIX = "app.security-analysis.";
```

Comment:
It is safer to separate all words by points to make it more compatible with other TSDBs than prometheus through the Observation API